packages/server/src/handlers/listCurrencies.ts

```diff
@@ -0,0 +1,61 @@
+import { Currency, CurrencyType } from "@ledgerhq/wallet-api-core";
+import type { SimpleJSONRPCMethod } from "json-rpc-2.0";
+import type { ServerParams } from "..";
+import globStringToRegex from "../utils/globStringToRegex";
+
+const listCurrencies: SimpleJSONRPCMethod<ServerParams> = (
+  params,
+  serverParams
+): Currency[] => {
+  if (!serverParams || !serverParams.currencies) {
+    return [];
+  }
+
+  const { currencies: serverCurrencies } = serverParams;
+
+  if (!params) {
+    return serverCurrencies;
+  }
+
+  const { params: methodParams } = params as {
+    params?: {
+      /**
+       * Include tokens in the results
+       */
+      includeTokens?: boolean;
+      /**
+       * Select a set of currencies by id. Globing is enabled
+       */
+      currencies?: string[];
+    };
+  };
+
+  if (!methodParams) {
+    return serverCurrencies;
+  }
+
+  const { includeTokens, currencies } = methodParams;
+
+  const listedCurrencies = includeTokens
+    ? serverCurrencies
+    : serverCurrencies.filter(
+        (currency) => currency.type === CurrencyType.CryptoCurrency
+      );
```

Comment:
Same question as with `serverAccounts`

Reply:
Updated in d7f1c562c40ddfcfa6ffe999222cd8b9da5dfc8b